.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
```

Comment:
7 is not a good version--odd-numbered releases are unsupported after 6 months.  I think we should be testing on 10 and 12, which are the current active releases.

Reply:
Done.